kuma/wiki/tests/test_helpers.py

```diff
@@ -15,6 +17,15 @@ def test_tojson(self):
         eq_(tojson({'title': '<script>alert("Hi!")</script>'}),
             '{"title": "&lt;script&gt;alert(&quot;Hi!&quot;)&lt;/script&gt;"}')
 
+    @mock.patch_object(Site.objects, 'get_current')
+    def test_absolutify(self, get_current):
+        get_current.return_value.domain = 'testserver'
+
+        eq_(absolutify('/woo'), 'https://testserver/woo')
+        eq_(absolutify('http://domain.com'), 'http://domain.com')
+        site = Site(domain='otherserver')
+        eq_(absolutify('/woo', site), 'https://otherserver/woo')
```

Comment:
Needs a test for `absolutify('')` and `absolutify('/')`, maybe even `absolutify('//')`